scripts/test/js_test.sh

```diff
@@ -11,7 +11,7 @@ elif [[ ! -z "$WATCH" ]]
 then
     export CMD="node ./node_modules/mocha/bin/_mocha --watch"
 else
-    export CMD="node ./node_modules/mocha/bin/_mocha"
+    export CMD="node ./node_modules/mocha/bin/_mocha --exit"
```

Comment:
JS tests are still timing out for some reason, this didn't seem to fix it?

Reply:
hm, I was hoping it would because it fixed it locally. I'll look into it and see

Reply:
ah, I forgot how the tests are called on travis. I think it should be good now (we'll see I suppose).

Reply:
ok looks like that fixed the test exit issue! build is still failing b/c of the sass linter though, oops

Reply:
haha, fixed the sass linter, ran into a thing with uglify...